extensions/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Isn't empty `.gitkeep` better?

Reply:
> .gitkeep isn’t documented, because it’s not a feature of Git.

https://stackoverflow.com/a/7229996

It is like a wide-spread fairy-tale.